spec/features/remember_device/cookie_expiration_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+describe 'signing in with remember device and closing browser' do
+  include SamlAuthHelper
+
+  let(:cookies) do
+    Capybara.
+      current_session.
+      driver.
+      browser.
+      current_session.
+      instance_variable_get(:@rack_mock_session).
+      cookie_jar.
+      instance_variable_get(:@cookies)
+  end
```

Comment:
I wouldn't put this in a `let` block... since lets get cached and this is a value to be modified